python/ray/_private/runtime_env.py

```diff
@@ -157,6 +97,87 @@ def serialize(self) -> str:
         return json.dumps(self._dict, sort_keys=True)
 
 
+def runtime_env_prep(runtime_env_json: dict):
+    result_dict = dict()
+    result_dict["conda"] = None
+    if "conda" in runtime_env_json:
+        if sys.platform == "win32":
+            raise NotImplementedError("The 'conda' field in runtime_env "
+                                      "is not currently supported on "
+                                      "Windows.")
+        conda = runtime_env_json["conda"]
+        if isinstance(conda, str):
+            yaml_file = Path(conda)
```

Comment:
You probably want to do `working_dir / yaml_file` if working dir is not None. the same for pip requirement

Reply:
I see, this is because this function might be called from a remote cluster, which doesn't have the local file--I overlooked this, good catch